modules/simple_forms_api/app/services/simple_forms_api/notification_email.rb

```diff
@@ -427,6 +425,13 @@ def form40_10007_first_name
       end
     end
 
+    def email_args
```

Comment:
I'm introducing `email_args` as a refactoring to make it more in line with the `form_upload_notification_email`.